ui/index.js

```diff
@@ -297,9 +297,6 @@ function setupStateHooks(store) {
     // for more info)
     state.version = global.platform.getVersion();
     state.browser = window.navigator.userAgent;
-    state.completeTxList = await actions.getTransactions({
```

Comment:
Reasons not entirely clear to me, but this appears to be from https://github.com/MetaMask/metamask-extension/pull/27611